info.puredata.Pd.yml

```diff
@@ -0,0 +1,74 @@
+app-id: info.puredata.Pd
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: pd-gui
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access
+  - --filesystem=home
+# MIDI, webcams, controllers,...
+  - --device=all
+modules:
+  - shared-modules/linux-audio/jack2.json
```

Comment:
```suggestion
```

You don't need jack here. The headers are in the SDK.

Reply:
thanks.fixing